stacks/set.go

```diff
@@ -0,0 +1,39 @@
+package stacks
+
+import (
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/go-scalingo"
+	"gopkg.in/errgo.v1"
+)
+
+func Set(app string, stack string) error {
+	c := config.ScalingoClient()
+	stacks, err := c.StacksList()
+	if err != nil {
+		return errgo.Notef(err, "fail to list available stacks")
+	}
+
+	var stackToSet scalingo.Stack
+	for _, apistack := range stacks {
```

Comment:
Is it mandatory to do all this? `AppsSetStack` should already test that the stack exists before it sets it, shouldn't it?

Reply:
It's to get the ID, because we can't set a stack by its name.

Reply:
OK but don't we want to update the `AppsSetStack` method to accept both ID and name, like for some other API endpoint?

Reply:
Hmmm are we doing this? For which endpoint, can you tell me?

Reply:
You're right, we don't do that. I supposed we do that for the apps endpoint in which we can use both the ID or the name but apparently no.